HTML-CSS-Assignments-master/src/Assignment-2/assignment2.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+	<meta http-equiv="Pragma" content="no-cache" />
+	<meta http-equiv="Expires" content="0" />
+	<script src="bundle.js" type="text/javascript"></script>
+</head>
+
+<body>
+	<div id="container">
+
+	<!-- Start your code from here -->
+	<div id="container">
+		<header class="page-header">
+			<div class="page-header-logo">
+				<img src="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.planetware.com%2Fpictures%2Findia-ind.htm&psig=AOvVaw3gU5LALSyIKXxQLNaD-MEX&ust=1636560056527000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCIC_gbvTi_QCFQAAAAAdAAAAABAN" height="70px" width="70px">
+			</div>
+
+			<form class="page-header-form">
+				<input type="text" placeholder="Search" name="Search">
+			</form>
+
+
+		</header>
+		<nav class="page-header-navigation">
+			<section class="Link1">
+				<a href="https://www.instagram.com" target="_blank">Instagram </a>
+			</section>
+
+			<section class="Link1">
+				<a href="https://www.the10xacademy.com/" target="_blank">The 10xacademy</a>
+			</section>
+
+			<section class="Link1">
+				<a href="https://www.facebook.com/" target="_blank">Facebook</a>
+			</section>
+
+			<section class="Link4">
+				<a href="https://www.geeksforgeekscom/" target="_blank">GeeksforGeeks</a>
+			</section>
+		</nav>
+
+		<section class="left_container">
+			<aside>
+				<ul>Section 1</ul>
+				<ul>Section 2</ul>
```

Comment:
the format using ul tag is

> 				<ul>
					<li>section 1</li>
					<li>section 2</li>
					<li>section 3</li>
					<li>section 4</li>
					<li>section 5</li>
					<li>section 6</li>
					<li>section 7</li>
					<li>section 8</li>
					<li>section 9</li>
					<li>section 10</li>
				</ul>